### ISSUE TITLE
Learning rate with keras

### ISSUE THREAD

**Issue by theabc123** [CSTR-Edinburgh/merlin#384]
I switched to Keras backend, and I noticed that changing the learning_rate in the config files has no effect on the training process (tried 0.000001 to 0.9). I always get the exact same validation error values for the different epochs.
How can I pass a given learning_rate value to Merlin when I use ? 
[Architecture]
hidden_layer_size  : [ 2048, 1024, 512]
hidden_layer_type  : [ 'BLSTM', 'BLSTM',  'BLSTM']
switch_to_tensorflow: False
switch_to_keras: True

**Comment by m-toman** [comment]
That's because the learning rate isn't passed to the optimizer here https://github.com/CSTR-Edinburgh/merlin/blob/master/src/keras_lib/model.py#L206

**Comment by theabc123** [comment]
@m-toman  thanks a lot, I came to the same conclusion, for now I changed line 70 in src/keras_lib/model.py :        
self.optimizer     = keras.optimizers.Adam(lr=0.00002, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False) 

And it works.

**Comment by m-toman** [comment]
Yes, I did the same. But as that ignores the "optimizer" configuration setting I never made a PR :).
I've also attached https://github.com/bckenstler/CLR with good results.
